packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -60,6 +62,16 @@ const defaultPermissions = { canManageProject: true };
 const mergeOptional = (defaultValue, value) =>
   value === null ? undefined : { ...defaultValue, ...value };
 
+const ApolloProvider = props =>
+  props.mocks ? (
+    <ApolloMockProvider {...props} />
+  ) : (
+    <ApolloActualProvider client={createApolloClient()}>
```

Comment:
I did not know that `createApolloClient` returns a closure.
Because in `configure-apollo`, you exported `createApolloClient` already being invoked.

```js
export default createApolloClient();
```